sensor_msgs/Image image_raw
string dataset
float32 confidence
float32 nms
---
lasr_perception_server/Detection[] detected_objects